packages/@roots/sage/src/Sage/index.ts

```diff
@@ -14,40 +9,8 @@ const inDevelopment = (app: Framework) => {
   app.devtool()
 
   app.extensions.get('@roots/bud-entrypoints').setOptions({
-    publicPath: '',
+    publicPath: 'auto',
```

Comment:
@QWp6t no matter what this is set to it seems to use the publicPath specified via webpack. probably a logic fail somewhere in the plugin.